config/rbac/role_binding.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: watermarkpodautoscaler-controller-manager
+  name: watermarkpodautoscaler-manager
```

Comment:
is it a manual change you did?

Reply:
yes, should it be done somewhere else?

Reply:
I think for this one is manual, but it can be referenced by some kustomize logic. I checked and I did found anything.
So all good 👍